pkg/kubelet/eviction/helpers.go

```diff
@@ -779,7 +779,7 @@ func thresholdsMet(thresholds []evictionapi.Threshold, observations signalObserv
 		threshold := thresholds[i]
 		observed, found := observations[threshold.Signal]
 		if !found {
-			glog.Warningf("eviction manager: no observation found for eviction signal %v", threshold.Signal)
+			glog.V(4).Infof("eviction manager: no observation found for eviction signal %v", threshold.Signal)
```

Comment:
I actually think we should leave this, and the one below as `Warningf`.  These are important signals to users that their stats arent working.